pkg/catalog/service.go

```diff
@@ -9,21 +11,27 @@ import (
 func (mc *MeshCatalog) GetServicesForServiceAccount(sa service.K8sServiceAccount) ([]service.MeshService, error) {
 	var services []service.MeshService
 	for _, provider := range mc.endpointsProviders {
+
 		// TODO (#88) : remove this provider check once we have figured out the service account story for azure vms
-		if provider.GetID() != constants.AzureProviderName {
-			log.Trace().Msgf("[%s] Looking for Services for Name=%s", provider.GetID(), sa)
-			providerServices, err := provider.GetServicesForServiceAccount(sa)
-			if err != nil {
-				log.Warn().Msgf("Error getting services from provider %s: %s", provider.GetID(), err)
-			} else {
-				log.Trace().Msgf("[%s] Found services %v for Name=%s", provider.GetID(), services, sa)
-				services = append(services, providerServices...)
+		if provider.GetID() == constants.AzureProviderName {
```

Comment:
Should we just remove AzureProvider from `endpointProviders` for the time being instead?

I think it'll look cleaner overall, and once we retake the VM endpoints effort we can always add it back

Reply:
@eduser25 - we could inded remove the Azure Endpoint Provider stuff until we have Identity + VM representation in SMI.